src/equations/laplace_diffusion_1d.jl

```diff
@@ -0,0 +1,50 @@
+@doc raw"""
+    LaplaceDiffusion1D(diffusivity, equations)
+
+`LaplaceDiffusion1D` represents a scalar diffusion term ``\nabla \cdot (\kappa\nabla u))``
+with diffusivity ``\kappa`` applied to each solution component defined by `equations`.
+"""
+struct LaplaceDiffusion1D{E, N, T} <: AbstractLaplaceDiffusion{1, N}
+  diffusivity::T
+  equations_hyperbolic::E
+end
+
+LaplaceDiffusion1D(diffusivity, equations_hyperbolic) =
+  LaplaceDiffusion1D{typeof(equations_hyperbolic), nvariables(equations_hyperbolic), typeof(diffusivity)}(diffusivity, equations_hyperbolic)
+
+varnames(variable_mapping, equations_parabolic::LaplaceDiffusion1D) =
+  varnames(variable_mapping, equations_parabolic.equations_hyperbolic)
+
+# no orientation specified since the flux is vector-valued
+function flux(u, gradients, orientation::Integer, equations_parabolic::LaplaceDiffusion1D)
+  dudx = gradients
+  if orientation == 1
+    return equations_parabolic.diffusivity * dudx
+  end
+end
```

Comment:
```suggestion
function flux(u, gradients, orientation::Integer, equations_parabolic::LaplaceDiffusion1D)
  dudx = gradients
  # orientation == 1
  return equations_parabolic.diffusivity * dudx
end
```
Otherwise, we would end up with a type instability (returning something or `nothing`, depending on the value of `orientation`).

Reply:
Thanks, I'll change that.